boa_ast/Cargo.toml

```diff
@@ -13,6 +13,7 @@ rust-version.workspace = true
 [features]
 serde = ["dep:serde", "boa_interner/serde", "bitflags/serde"]
 arbitrary = ["dep:arbitrary", "boa_interner/arbitrary", "num-bigint/arbitrary"]
+temporal = []
```

Comment:
I'd propose changing this for an "experimental" flag instead, so that we don't need to remove the "temporal" feature when it reaches stage 4. That would also allow us to provide more implementations for other stage 3 features without adding more flags for each one of them.